BUILD

```diff
@@ -0,0 +1,169 @@
+package(
+    default_visibility = ["//visibility:public"],
+    features = [
+        "-layering_check",
+        "-parse_headers",
+    ],
+)
+
+exports_files([
+    "LICENSE",
+    "build_defs",
+    "reflection/reflection.fbs",
+    "samples/monster.fbs",
```

Comment:
why is any of the tests and samples stuff exported?

Reply:
I think so you can reference the files for codelabs.  Normall, bazel doesn't let you reference files from other packages unless you export the files.

Reply:
Probably not needed for this case, so I'll remove.